fadecandy_driver/src/ros/fadecandy_driver_ros_node.cpp

```diff
@@ -0,0 +1,35 @@
+#include <ros/init.h>
+#include <ros/node_handle.h>
+
+#include "./fadecandy_driver_ros.h"
+
+int main(int argc, char** argv)
+{
+  ros::init(argc, argv, "fadecandy_driver");
+  ros::NodeHandle local_nh("~");
+  fadecandy_driver::FadecandyDriverRos Node;
+
+  double restart_patience = 1.;
+  while (ros::ok())
+  {
+    try
+    {
+      if (!Node.initialized)
+      {
+        ROS_INFO("Connecting to Fadecandy device ..");
+        Node.connect();
```

Comment:
connect is not a correct name, please call it spin or run or something, you are doing more than connecting